examples/docs_beta_snippets/docs_beta_snippets/guides/external-systems/databases/use-envvars.py

```diff
@@ -0,0 +1,79 @@
+import os
+
+import pandas as pd
+from dagster_snowflake import SnowflakeResource
+from snowflake.connector.pandas_tools import write_pandas
+
+import dagster as dg
+
+# highlight-start
+# Define `local` and `production` versions of the Snowflake resource
+resources = {
```

Comment:
I followed a pattern from our existing docs for this. Is this fine or is there a better way to do swaps?

Reply:
This is one way to do it, that I think works if you have different environment variables for dev, production, or branch deployments.

However, my preferred approach is to use a single `SnowflakeResource` with a var like `SNOWFLAKE_USER`, and that environment variable value being different depending on if you are in dev or production.

In this example, having them separate is needed as there are hard coded values, like `database="PRODUCTION"`, however, if all parameters were env vars,  then I think a single instance could be used.